framework/test/unit/engine/consensus/consensus.spec.ts

```diff
@@ -168,6 +168,12 @@ describe('consensus', () => {
 			expect(network.registerEndpoint).toHaveBeenCalledTimes(4);
 		});
 
+		it('should register handlers', async () => {
```

Comment:
maybe a bit more descriptive 
```suggestion
		it('should register network handlers', async () => {
```